views/_partials/header.jade

```diff
@@ -17,7 +17,7 @@
                 li: a(href="/bootlint/" + themeQs) Bootlint
                 li: a(href="/legacy/" + themeQs) Legacy
 
-            ul.nav.navbar-nav.navbar-right.visible-md.visible-lg
```

Comment:
@jmervine good call, it works on mobile.  Reason I removed that those classes is that it was hiding  `Showcase` and `Integrations`.  Is the way I did it hacky?